### PR TITLE
Issue_861

### LINKED ISSUE #861: dragged item is copied to all containers (only on previously saved covers)
when a cover is created, saved, and then re-edited, dragged items are copied to all containers

drag-drop

### PR DESCRIPTION
**Pull request by Mubra**
closes #861 
solve

### CODE REVIEW

**Comment by scmos** [comment]
I just tried the change and it works correctly

**Comment by idgserpro** [comment]
First of all, thanks for this PR. (Just out of curiosity, which software are you using to create the gifs?)

Anyways. For this PR, it's a good pratice to:

- Add your change to https://github.com/collective/collective.cover/blob/master/CHANGES.rst. Always add your change to the last unreleased version, at the top (read the file to understand the references to issues as well);
- Create a test for this implementation using robot framework (an example https://github.com/collective/collective.cover/blob/master/src/collective/cover/tests/test_basic_tile.robot - read it to understand a bit of the syntax). We know this may seem _overkill_, since your change is just one line: but it's important to have a test suite to avoid breakage.

**Comment by scmos** [comment]
> First of all, thanks for this PR. (Just out of curiosity, which software are you using to create the gifs?)
> 
> Anyways. For this PR, it's a good pratice to:
> 
> * Add your change to https://github.com/collective/collective.cover/blob/master/CHANGES.rst. Always add your change to the last unreleased version, at the top (read the file to understand the references to issues as well);
> * Create a test for this implementation using robot framework (an example https://github.com/collective/collective.cover/blob/master/src/collective/cover/tests/test_basic_tile.robot - read it to understand a bit of the syntax). We know this may seem _overkill_, since your change is just one line: but it's important to have a test suite to avoid breakage.


- We are using https://www.screentogif.com/ for animated GIFs.
- The CHANGE.rst file we forget to update it, we will update it.
- The test in robot we are omitting because we do not know how to use it, we will review the tutorial to implement it in the future.

**Comment by Mubra** [comment]
Hi @idgserpro , the program I use is called 'ScreentoGif', I hope not to cause a problem, but if you like, I leave the link here https://www.screentogif.com/.

I will take these comments into account for my next PR

**Comment by idgserpro** [comment]
@Mubra, did @hvelarde  invite you to the Collective Organization? If so, you can create branchs directly in `collecive/collective.cover`.

**Comment by Mubra** [comment]
Ready, accept @hvelarde's invitation

**Comment by idgserpro** [comment]
Also don't forget to put your userid under the entry.

**Comment by Mubra** [comment]
@idgserpro  sorry, which 'ID' do you mean?

**Comment by idgserpro** [comment]
> @idgserpro sorry, which 'ID' do you mean?

[Mubra]

see:

https://raw.githubusercontent.com/collective/collective.cover/master/CHANGES.rst

File: CHANGES.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ There's a frood who really knows where his towel is.
 2.2.1 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
-- solve issue #861.
+- The 'drop' event and its 'target' method are used to obtain the element (column) and not generate ambiguity (fixes # 861). [Mubra]
```
Comment (idgserpro) [inline CHANGES.rst]: use:

(fixes \`#861 \<https://github.com/collective/collective.cover/issues/861>`_)

to link to issue.

see version 2.1b2 in:

https://raw.githubusercontent.com/collective/collective.cover/master/CHANGES.rst

Comment (Mubra) [reply CHANGES.rst]: thanks for the support @idgserpro

**Pull request merged**